dvc/progress.py

```diff
@@ -62,18 +65,30 @@ def __init__(
         desc  : persists after `close()`
         level  : effective logging level for determining `disable`;
             used only if `disable` is unspecified
+        disable  : If (default: None), will be determined by logging level.
+            May be overridden to `True` due to non-TTY status.
+            Skip override by specifying env var `DVC_ISATTY`.
         kwargs  : anything accepted by `tqdm.tqdm()`
         """
         kwargs = kwargs.copy()
         kwargs.setdefault("unit_scale", True)
+        kwargs.setdefault("file", sys.stderr)
```

Comment:
Why do we now set "file" into the kwargs anyway?

Reply:
good point; not needed